daily_open_prs.rb

```diff
@@ -51,49 +53,48 @@
 pr_cache = []
 
 repos.each do |repo|
-
-  #Retrieves all PRs for the repo
-  pr_cache = (util.fetch_async("#{options[:namespace]}/#{repo}", search_with={:state => 'open'}, filter=[]))
-  pr_cache.concat(util.fetch_async("#{options[:namespace]}/#{repo}", search_with={:state => 'closed'}, filter=[]))
+  # Retrieves all PRs for the repo
+  pr_cache = util.fetch_async("#{options[:namespace]}/#{repo}", search_with, filter)
```

Comment:
Why are the options[:namespace] and {repo} used here, shouldn't this be the parsed data from the modules.json ?